src/sentry/static/sentry/app/routes.jsx

```diff
@@ -164,6 +164,23 @@ const accountSettingsRoutes = [
     componentPromise={() => import('./views/settings/account/accountAuthorizations')}
     component={errorHandler(LazyLoad)}
   />,
+
+  <Route key="api" path="api/" name="API">
+    <Route path="applications/" name="Applications">
+      <IndexRoute
+        componentPromise={() =>
+          import(/*webpackChunkName: "ApiApplications"*/ './views/settings/account/apiApplications')}
+        component={errorHandler(LazyLoad)}
+      />
+      <Route
+        path=":appId/"
+        name="Details"
+        componentPromise={() =>
+          import(/*webpackChunkName: "ApiApplicationDetails"*/ './views/settings/account/apiApplicationDetails')}
+        component={errorHandler(LazyLoad)}
```

Comment:
is this being chosen to be lazy loaded based on size or mostly infrequency?

Reply:
Mostly infrequency, (but atm could be b/c of size if it uses forms)